setup.py

```diff
@@ -73,7 +73,7 @@
         "eth-account>=0.5.2,<0.6.0",
         "eth-hash[pycryptodome]>=0.2.0,<1.0.0",
         "eth-typing>=2.0.0,<3.0.0",
-        "eth-utils>=1.8.4,<2.0.0",
+        "eth-utils>=1.8.4,<1.9.1",
```

Comment:
FYI: the revert message work will require a later eth-utils version. Without it, EthereumTesterProvider won't receive revert messages. Can be figured out later.